source/reference/method/MongoDBCollection-replaceOne.txt

```diff
@@ -90,6 +88,14 @@ Parameters
 
          .. versionadded:: 1.3
 
+     * - sort
+       - array|object
+       - The sort specification for the ordering of the matched
+         documents. Set this option to apply an order to matched
+         documents before the driver performs the replace operation.
```

Comment:
The driver is only sending the option to the server. There is no logic in the driver to implement this logic. So I suggest replacing "driver" with ,"server" here, and in the 2nd page.
```suggestion
         documents before the server performs the replace operation.
```